### DISCUSSION TITLE
integrate imagepullsecrets with secretsmanager

### DISCUSSION THREAD

**Discussion by gbucknel** [bottlerocket-os/bottlerocket#2351] [Q&A]
Hi,

I'd really like to add imagePullSecrets to the EKS node configuration , as in here: 

https://github.com/bottlerocket-os/bottlerocket#container-image-registry-settings

I'm trying to enable a local caching registry. 
Is there any way to integrate this configuration with Secrets Manager ? The ImagePullSecrets seem to be available in plaintext if you go to the LaunchTemplate in EC2, which makes me pause a bit. 

Thank you so much,
Graham.

**Comment by mchaker** [comment]
Thanks for bringing this to our attention! I've gone ahead and opened an issue for this: #2355

**Reply by gbucknel** [reply]
@mchaker Thanks for looking at it! :)
